test/test_indexing.py

```diff
@@ -105,6 +105,26 @@ def test_int_indices(self, device):
         self.assertEqual(v[:, [0, 4, 2]].shape, (5, 3, 3))
         self.assertEqual(v[:, [[0, 1], [4, 3]]].shape, (5, 2, 2, 3))
 
+    @dtypesIfCPU(torch.float, torch.long, torch.bool, torch.bfloat16)
+    @dtypesIfCUDA(torch.half, torch.long, torch.bool)
```

Comment:
I think you need to have @dtypes(...), before you have the more specific versions here. @dtypes(...) should describe types you expect to run on any device type. In this case maybe torch.bool and torch.long?